db/schema.rb

```diff
@@ -570,6 +595,138 @@
      FROM (appointments ap
        JOIN facilities f ON ((ap.facility_id = f.id)));
   SQL
+  create_view "latest_blood_pressures_per_patient_per_days", materialized: true, sql_definition: <<-SQL
```

Comment:
Should all these views be in the diff? /cc https://github.com/simpledotorg/simple-server/pull/939